src/main/java/bio/terra/cli/command/config/set/Logging.java

```diff
@@ -13,10 +13,10 @@
   Logging.LogLevelArgGroup argGroup;
 
   static class LogLevelArgGroup {
-    @CommandLine.Option(names = "--console", description = "console logging level")
+    @CommandLine.Option(names = "--console", description = "Console logging level.")
     private boolean console;
 
-    @CommandLine.Option(names = "--file", description = "file logging level")
+    @CommandLine.Option(names = "--file", description = "File logging level.")
```

Comment:
Maybe indicate where the log files live here.

Reply:
When an error occurs, the log file path is printed out to the console. I could see possibly adding a separate command to get the log file path, but I don't think adding it to the config command for the log level is an intuitive place to look for it.